modules/jpg/resource_saver_jpege.cpp

```diff
@@ -0,0 +1,146 @@
+/*************************************************************************/
+/*  resource_saver_jpege.cpp                                             */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2022 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2022 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "resource_saver_jpege.h"
+
+#include "core/io/file_access.h"
+#include "scene/resources/texture.h"
+
+void ResourceSaverJPG::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("save_image", "path", "image"), &ResourceSaverJPG::save_image);
+	ClassDB::bind_method(D_METHOD("save_image_to_buffer", "image"), &ResourceSaverJPG::save_jpg_to_buffer);
+	ClassDB::bind_method(D_METHOD("set_encode_options", "options"), &ResourceSaverJPG::set_encode_options);
+	BIND_ENUM_CONSTANT(SUBSAPLING_Y_ONLY);
+	BIND_ENUM_CONSTANT(SUBSAPLING_H1V1);
+	BIND_ENUM_CONSTANT(SUBSAPLING_H2V1);
+	BIND_ENUM_CONSTANT(SUBSAPLING_H2V2);
+	BIND_ENUM_CONSTANT(SUBSAPLING_MAX);
```

Comment:
Typo: It should be "SUBSAMPLING" :slightly_smiling_face:

Reply:
:deciduous_tree:

Reply:
![subsapling.jpg](https://user-images.githubusercontent.com/4701338/166109889-1517ccf5-ba72-4bfe-a8be-e762d4b77837.png)